test/uk/gov/hmrc/pensionschemereturn/postman/LocalHost.postman_environment.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you please put few lines in readme in this directory about how to use environments in postman, assuming not everyone used that before.

Reply:
readme updated with postman explanations